app/models/user.rb

```diff
@@ -1,12 +1,18 @@
 class User < ApplicationRecord
-  has_many :posts
-  has_many :likes
-  has_many :comments
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  # rubocop:disable ...
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  # rubocop:enable ...
+  has_many :posts, foreign_key: 'user'
+  has_many :likes, foreign_key: 'user'
+  has_many :comments, foreign_key: 'user'
```

Comment:
- [ ] [OPTIONAL] I wanted to know what you mean here, do you mean that you have a field (attribute) in the user's table called `user`? I was expecting you to use a primary key as a foreign key, as per the requirement.